routers/api/v1/repo/file.go

```diff
@@ -17,7 +17,7 @@ import (
 	"code.gitea.io/gitea/modules/repofiles"
 	api "code.gitea.io/gitea/modules/structs"
 	"code.gitea.io/gitea/modules/web"
-	"code.gitea.io/gitea/routers/repo"
+	"code.gitea.io/gitea/routers/web/repo"
```

Comment:
if we have to import web router package, we probably have a function witch we should move int services ...